symphony-bdk-core/src/test/java/com/symphony/bdk/core/util/PresentationMLParserTest.java

```diff
@@ -0,0 +1,38 @@
+package com.symphony.bdk.core.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.symphony.bdk.core.util.exception.PresentationMLParserException;
+
+import org.junit.jupiter.api.Test;
+
+public class PresentationMLParserTest {
+
+  @Test
+  void getMessageFromPresentationMLTest() throws PresentationMLParserException {
+    String presentationML = "<div data-format=\"PresentationML\" data-version=\"2.0\"> \n"
+        + "  <a href=\"http://www.symphony.com\">This is a link to Symphony's Website</a> \n"
+        + "</div>";
+
+    String content = PresentationMLParser.getMessageTextContent(presentationML);
+
+    assertEquals(content.trim(), "This is a link to Symphony's Website");
```

Comment:
Interesting that you have to `trim()`. Isn't that something that we could add `PresentationMLParser#getTextContent(String presentationML, Boolean trim)`? 

We would still `PresentationMLParser#getTextContent(String presentationML)` we `trim` set to `true` by default

Reply:
the asserts could be just assertEquals(" This is a link to Symphony's Website ") right?

Reply:
there's also line break at two ends of sentences